This was the error message:
```
cargo:warning=deps/thorvg\src/renderer/sw_engine\tvgSwRenderer.cpp:37:8: error: unknown type name 'mutex'
cargo:warning=   37 | static mutex _rendererMtx;
```

I'll see about changing the if condition

--- a/src/renderer/cpu_engine/tvgSwRenderer.cpp
+++ b/src/renderer/cpu_engine/tvgSwRenderer.cpp
@@ -34,7 +34,11 @@
 /************************************************************************/
 
 static int32_t _rendererCnt = -1;
+#ifdef __STDCPP_THREADS__
 static mutex _rendererMtx;
+#else
+static struct { void lock() {} void unlock() {} } _rendererMtx;
+#endif
 
 struct SwTask : Task
 {

--- a/src/renderer/gpu_engine/gl/tvgGlRenderer.cpp
+++ b/src/renderer/gpu_engine/gl/tvgGlRenderer.cpp
@@ -36,7 +36,11 @@
 #define NOISE_LEVEL 0.5f
 
 static int32_t _rendererCnt = -1;
+#ifdef __STDCPP_THREADS__
 static mutex _rendererMtx;
+#else
+static struct { void lock() {} void unlock() {} } _rendererMtx;
+#endif
 
 static constexpr float IDENTITY_VERTEX[] = {-1.f, 1.f, -1.f, -1.f, 1.f, 1.f, 1.f, -1.f};
 static constexpr uint32_t RECT_INDEX[] = {0, 1, 2, 2, 1, 3};

--- a/src/renderer/gpu_engine/wg/tvgWgRenderer.cpp
+++ b/src/renderer/gpu_engine/wg/tvgWgRenderer.cpp
@@ -28,7 +28,11 @@
 /************************************************************************/
 
 static int32_t _rendererCnt = -1;
+#ifdef __STDCPP_THREADS__
 static mutex _rendererMtx;
+#else
+static struct { void lock() {} void unlock() {} } _rendererMtx;
+#endif
 
 
 void WgRenderer::release()

